regl-camera.js

```diff
@@ -41,10 +42,21 @@ function createCamera (regl, props_) {
   var prevY = 0
 
   if (isBrowser && props.mouse !== false) {
-    mouseChange(function (buttons, x, y) {
+    var source = element || window
+
+    function getWidth () {
+      return element ? element.offsetWidth : window.innerWidth
+    }
+
+    function getHeight () {
+      return element ? element.offsetHeight : window.innerHeight
+    }
+
+    var noScroll = !!props.noScroll
```

Comment:
Reverted behaviour to as it was before (although I still wonder if it should just always set to `true`)